button-examples/iOS/ios-button-example.md

```diff
@@ -19,7 +19,9 @@ There are four types of buttons:
 1. [Text button](#text-button)
 2. [Outlined button](#outlined-button)
 3. [Contained button](#contained-button)
-4. [Toggle button](#toggle-button)
+4. [Toggle button](#toggle-button) - Not Yet Supported on iOS.
```

Comment:
Unfortunately, we're using the design article's hero examples, and that includes the hero images. And we're also following the terminology from the design article, which lumps the cards with icon buttons with the toggle bar.

@barbaraeldredge @ajsecord should we include a note stating that the icon button is supported, but not the toggle bar? I think we've discussed this in the past and at one point decided to drop the note.

Reply:
If engineers are our target our audience, then the doc must reflect the actual functionality offered by the platform:  
- There is no toggle buttons support on iOS.
- Card's toggle button look very different than the toggle buttons's image that's on this page.
- I'm not sure we should should show this image altogether, because it will mislead readers to believe this feature exists on iOS.